app/assets/javascripts/product_helper.js

```diff
@@ -1,4 +1,5 @@
 // Used to show the selected product and cost infomation in 'Select Product' step
```

Comment:
This file doesn't seem related to the PR, and I think it makes the JS less safe - changing constants to variables, for example

Reply:
Ah I've just read the commit message!